pkgs/build-support/writers/default.nix

```diff
@@ -178,13 +178,20 @@ rec {
   writeJSBin = name:
     writeJS "/bin/${name}";
 
+  awkFormatNginx = builtins.toFile "awkFormat-nginx.awk" ''
```

Comment:
Can we remove this now given we have: https://github.com/slomkowski/nginx-config-formatter/issues/16